chains-examples/docs/whisper-vad/vad_transcribe.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+import logging
+import time
+from pathlib import Path
+
+import asr_workers.whisper
+import truss_chains as chains
+import utilities.data_types as data_types
+import utilities.helpers as helpers
+import utilities.utilities as utilities
+
+# Configure the logging
+logging.basicConfig(level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+
+@chains.mark_entrypoint
+class TranscribeWithVad(chains.ChainletBase):
+    """Transcribes one file end-to-end and sends results to webhook."""
+
+    remote_config = chains.RemoteConfig(
+        docker_image=chains.DockerImage(
+            apt_requirements=["ffmpeg"],
+            # TODO: pin the version of the package
+            pip_requirements=[
+                "pandas",
+                "google-auth",
+                "google-cloud-bigquery",
+                "torch",
+                "torchaudio",
+                "toml",
+                "soundfile==0.10.2",
+            ],
+        ),
+        compute=chains.Compute(predict_concurrency=128),
+    )
+    _context: chains.DeploymentContext
+
+    _whisper: asr_workers.whisper.WhisperModel
+
+    def __init__(
+        self,
+        whisper: asr_workers.whisper.WhisperModel = chains.depends(
+            asr_workers.whisper.WhisperModel, retries=2
```

Comment:
The retries here can cause a lot of delays in legit failures. I'd recomend reducing them or turning them off by default.